test/hooks-pagination.ts

```diff
@@ -0,0 +1,106 @@
+import test from 'ava';
```

Comment:
Move the content of this file either to `test/hooks.ts` or `test/pagination.ts`

Reply:
Done, moved it into `test/hooks.ts` 🔥
Everything looks good now, just adding one more test to make sure intentional duplicate hooks still work.